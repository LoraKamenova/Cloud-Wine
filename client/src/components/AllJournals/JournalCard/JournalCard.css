.journal-card {
    background-color: white;
    width: 320px;
    height: 400px ;
    padding: 20px 20px 40px 20px;
    margin: 10px;
}

.journal-card-image {
    margin: 0 auto;
    width: 280px;
    height: 260px;
}

.journal-card-title-wrapper {
    font-size: 25px;
    font-family: 'Caveat', cursive;
    text-align: center;
    margin-top: 15px;
}
