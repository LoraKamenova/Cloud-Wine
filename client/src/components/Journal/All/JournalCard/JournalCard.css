.journal-card {
    width: 320px;
    height: 400px;
    background-color: white;
    margin: 10px;
    padding: 20px 20px 40px 20px;
    box-shadow: 0px 0px 8px -3px rgba(0,0,0,0.73);
}

.journal-card-image {
    width: 280px;
    height: 260px;
    margin: 0 auto;
}

.journal-card-title-wrapper {
    text-align: center;
    font-family: 'Caveat', cursive;
    font-size: 23px;
    margin-top: 15px;
}
