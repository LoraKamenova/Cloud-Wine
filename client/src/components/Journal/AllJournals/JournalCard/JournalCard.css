.journal-card {
    background-color: white;
    width: 320px;
    height: 400px ;
    padding: 20px 20px 40px 20px;
    margin: 10px;
    box-shadow: 0px 0px 8px -3px rgba(0,0,0,0.73);
}

.journal-card-image {
    margin: 0 auto;
    width: 280px;
    height: 260px;
}

.journal-card-title-wrapper {
    font-size: 23px;
    font-family: 'Caveat', cursive;
    text-align: center;
    margin-top: 15px;
}
