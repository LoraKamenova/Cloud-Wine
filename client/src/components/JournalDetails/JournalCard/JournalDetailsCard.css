
.journal-details-card {
    display: flex;
    flex-direction: column;
    justify-content: center;
}

.journal-details-card-header {
    text-align: center;
}

.journal-details-card-heading {
    font-size: 35px;
}

.journal-details-card-image-wrapper {
    display: flex;
    justify-content: space-around;
    margin: 20px 0;
}

.journal-details-card-image {
    width: 420px;
    height: 250px;
    box-shadow: 0px 0px 11px -3px #000000;
}

.journal-details-card-text {
    padding: 20px 40px;
    text-align: justify;
}

.journal-details-card-button-wrapper {
    display: flex;
    justify-content: flex-end;
}
