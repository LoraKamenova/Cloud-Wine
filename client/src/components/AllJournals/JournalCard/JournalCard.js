import './JournalCard.css'

const JournalCard = (props) => {
    return (
        <article className="journal-card">
            <div className="journal-card-image-wrapper">
                <img className="journal-card-image"
                     src={props.url}/>
            </div>
            <div className="journal-card-title-wrapper">
                <h3 className="journal-card-title">{props.title}</h3>
            </div>
        </article>
    );
};

export default JournalCard;
