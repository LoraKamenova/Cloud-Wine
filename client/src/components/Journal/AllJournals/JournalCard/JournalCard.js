import './JournalCard.css'

const JournalCard = (props) => {
    return (
        <article className="journal-card">
            <div className="journal-card-image-wrapper">
                <img className="journal-card-image"
                     src={props.imageUrl1} alt="journal"/>
            </div>
            <div className="journal-card-title-wrapper">
                <h3 className="journal-card-title">{props.title}</h3>
            </div>
        </article>
    );
};

export default JournalCard;
