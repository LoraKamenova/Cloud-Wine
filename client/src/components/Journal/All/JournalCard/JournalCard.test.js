import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import JournalCard from './JournalCard';

describe('JournalCard Component', () => {
    it('Should display entry title', () => {
        render(
            <BrowserRouter>
                <JournalCard title="Незабравима ваканция" link="/"/>
            </BrowserRouter>
        );

        expect(document.querySelector(".journal-card-title").textContent).toBe('Незабравима ваканция');
    });

    it("JournalCard Component snapshot", () => {
        const { asFragment } = render(<BrowserRouter><JournalCard link="/"/></BrowserRouter>);
        expect(asFragment()).toMatchSnapshot();
    });

});
