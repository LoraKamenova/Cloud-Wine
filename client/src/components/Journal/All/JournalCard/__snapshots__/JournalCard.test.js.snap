// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JournalCard Component JournalCard Component snapshot 1`] = `
<DocumentFragment>
  <article
    class="journal-card"
  >
    <div
      class="journal-card-image-wrapper"
    >
      <img
        alt="journal"
        class="journal-card-image"
      />
    </div>
    <div
      class="journal-card-title-wrapper"
    >
      <h3
        class="journal-card-title"
      />
    </div>
  </article>
</DocumentFragment>
`;
